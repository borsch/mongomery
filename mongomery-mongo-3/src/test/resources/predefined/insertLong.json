{
  "TestCollection": [
    {
      "stringField": "some value",
      "longField": "$insertLong(215345325523512)"
    }
  ]
}