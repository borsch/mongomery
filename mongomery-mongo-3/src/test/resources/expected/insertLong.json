{
  "TestCollection": [
    {
      "_id": "$anyObject()",
      "stringField": "some value",
      "longField": "$eqLongValue(215345325523512)"
    }
  ]
}